.github/workflows/lint.yml

```diff
@@ -238,5 +238,5 @@ jobs:
       env:
         CLANG_FORMAT: clang-format-10
       run: |
-        make -C hw/system/occamy all
-        util/git-diff.py --error-msg "::error ::Found differences, run \`make -C hw/system/occamy all\` before committing."
+        make -C hw/system/occamy update-rtl
+        util/git-diff.py --error-msg "::error ::Found differences, run \`make -C hw/system/occamy update-rtl\` before committing."
```

Comment:
Is there a specific reason that you removed the SW parts? I think it would also be good to check,e .g., the generated SW headers for differences.